docs/04-documentation.md

```diff
@@ -7,40 +7,152 @@ This page explains how to update documentation directly on GitHub.
+
+![Create a new branch for documentation updates]({{ path
+'/assets/screenshots/create-new-branch.gif' }})
+
+</details>
+
+## Step 3: Open documentation file for editing
+
+Ensure that you have switched to a new branch which is a good place to suggest
+changes. Then click on the button to open the README.md file in that component
+(example: `framework/components/ecl-footers/README.md`)
+
+<details><summary>_See how to do this step_</summary>
+
+![Edit documentaiton file]({{ path
+'/assets/screenshots/edit-documentation-file.gif' }})
+
+</details>
+
+## Step 4: Edit the documentation
+
+-   Make any modification you wish by using markdown editor. Here is a
+  [cheatsheet for markdown syntax](https://guides.github.com/features/mastering-markdown/)
+
+-   You can see result directly by switching to the "Preview" tab
+
+<details><summary>_See how to do this step_</summary>
+
+![Preview changes]({{ path '/assets/screenshots/edit-preview-changes.gif' }})
+
+</details>
+
+## Step 5: Save updates to the branch
+
+-   Once you have modified the file, you have to commit changes to save them in
+  your branch.
+
+-   Enter title and description based on
+  [naming conventions](https://ec-europa.github.io/europa-component-library/docs/conventions/git).
+
+-   Click on "Commit directly to the
+  `docs/update-{component-name}-{ticket-number}`".
+
+-   This will save changes on your branch.
+
+<details><summary>_See how to do this step_</summary>
+
+![Save changes to branch]({{ path
+'/assets/screenshots/save-changes-on-branch.gif' }})
+
+</details>
+
+## Step 6: Open a pull request
+
+-   Once make sure you have saved changes on your particular branch, you are ready
+  to proceed on proposing these changes to the main `master` branch which is
+  where you see production documentation.
+
+-   You can simply use the hints which GitHub gives you when you go to the root of
+  the repo after saving your changes.
+
+-   Do not forget to add `tag: documentation` on your pull request which is used
+  for the release log.
+
+<details><summary>_See how to do this step_</summary>
+
+![Open a pull request]({{ path '/assets/screenshots/open-pull-request.gif' }})
+
+</details>
+
+## Step 7: Preview your changes and follow-up
+
+-   When the pull request is open, our `HAL-Patch-INNO` bot will provide you a
+  preview of the ECL style guide with the updates provided by you.
+
+-   You can review them and make further changes on the branch repeating steps
+  1-5. Once the changes are saved, pull request is automatically updated, no
+  need to open another pull request.
+
+-   Changes on the pull request update the preview automatically.
+
+<details><summary>_See how to do this step_</summary>
+
+![Preview changes]({{ path '/assets/screenshots/preview-changes.gif' }})
+
+</details>
 
 ## It's done!
-Your pull request will be handled by our team and integrated to master ECL branch.  
+
+Your pull request will be briefly reviewed by our team that there are no linting
```

Comment:
Long phrase, commas:

"Your pull request will be briefly reviewed by our team, **so** that there are no linting issues and if all checks are green, changes will be integrated to **the** master ECL branch in short time.